components/places/src/storage/history.rs

```diff
@@ -224,10 +306,28 @@ pub fn delete_visits_since_in_tx(db: &impl ConnExt, since: Timestamp) -> Result<
         },
     )?;
 
+    // Insert tombstones for the deleted visits. We can't use each_chunk_mapped
+    // because we're inserting multiple values.
+    // TODO: Come up with a version of each_chunk that supports this case.
+    for chunk in visits.chunks(sql_support::default_max_variable_number() / 2) {
+        let sql = format!(
+            "INSERT INTO moz_historyvisit_tombstones(place_id, visit_date) VALUES {}",
```

Comment:
Hmm, since both `place_id` and `visit_date` are integers, I think we can build up the values string directly, and avoid chunking and binding params.